chart/kubeapps/README.md

```diff
@@ -110,9 +69,11 @@ Alternatively, a YAML file that specifies the values for parameters can be provi
 $ helm install --name kubeapps --namespace kubeapps -f custom-values.yaml bitnami/kubeapps
 ```
 
+## Configuration and installation details
+
 ### Configuring Initial Repositories
 
-By default, Kubeapps will track the [community Helm charts](https://github.com/helm/charts) and the [Kubernetes Service Catalog charts](https://github.com/kubernetes-incubator/service-catalog). To change these defaults, override the `apprepository.initialRepos` object:
+By default, Kubeapps will track the [community Helm charts](https://github.com/helm/charts) and the [Kubernetes Service Catalog charts](https://github.com/kubernetes-incubator/service-catalog). To change these defaults, override the `apprepository.initialRepos` object and install the chart using this custom file:
 
 ```console
 $ cat > custom-values.yaml <<EOF
```

Comment:
it doesn't make sense to generate a file with the command line if we are not specifying how to use it. I would rather not include anything and point to the `values.yaml` again.